anthos-bm-gcp-terraform/versions.tf

```diff
@@ -15,7 +15,7 @@
  */
 
 terraform {
-  required_version = ">= v0.15.5, < 1.2"
+  required_version = ">= v0.9.0, < 1.2"
```

Comment:
@apeabody  It's still doing the same thing

Reply:
@Shabirmean Curious - We might need to run release-please manually to understand which updater is being triggered.  But you can certainly roll-back this version change as it's only adding a minimum version.

Reply:
@Shabirmean @apeabody I believe this is something we need to improve in our TF updater for release please. Currently it matches any semver in versions.tf.

https://github.com/googleapis/release-please/blob/3248a06ee1769f43319ad49a1f7185416277dcce/src/updaters/terraform/module-version.ts#L28-L35

We could update it to match lines that start with `module_name` followed by semver.

As a workaround till then, you can directly edit this PR in the UI and revert that line. Release please will not force push until another commit is merged to main.

Reply:
Thanks @bharathkkb!

Reply:
Thank you very much @bharathkkb for explaining what's happening here. Also thank you for that suggestion. I have adopted that to ensure this release-pr can be shipped.

However, it might be best to have it addressed, either by fixing the releaser or by reverting this mainly because not all maintainers will be aware of this workaround when a release PR is openned